docs/docs/conceptual/choosing-a-cms.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Choosing Your CMS(s)
+---
+
+At their core, most medium-sized and large Gatsby websites are backed by a so-called "headless CMS". Headless CMSes excel at allowing content editors to flexibly create a content schema of inter-related content models.
+
+One of the key questions people ask when building a Gatsby site is "which CMS should I use?"
+
+## Choosing a Headless CMS
+
+If you're looking for a primary CMS (general-purpose, flexible content modelling) to power your website, there are several ways you could make your decision:
+
+**First**, focus on **top-flight general-purpose CMSs** with **first-class Gatsby integrations** and **high popularity** among Gatsby users. A ["first-class integration"](https://support.gatsbyjs.com/hc/en-us/articles/360052503494-Developing-a-first-class-CMS-integration-for-Gatsby-Cloud) means it supports the main Cloud functionality of Gatsby -- previews and incremental builds.
```

Comment:
For scan-ability, I'd add in h3s for the three major points in this section (popular integrations, price, and project requirements).

So like this one could be:

```
### 1) Popular, first-class Gatsby integrations

**First**, focus on...
```

Reply:
Ah, that's a great point. I've done this!